config/jobs/cert-manager/cert-manager/cert-manager-periodics.yaml

```diff
@@ -555,3 +555,363 @@ periodics:
 #         requests:
 #           cpu: 3500m
 #           memory: 12Gi
+#
+- name: ci-cert-manager-e2e-feature-gates-disabled-v1-18
+  interval: 24h
+  agent: kubernetes
+  decorate: true
+  extra_refs:
+  - org: cert-manager
+    repo: cert-manager
+    base_ref: master
+  annotations:
+    testgrid-create-test-group: 'true'
+    testgrid-dashboards: jetstack-cert-manager-master
+    testgrid-alert-email: cert-manager-dev-alerts@googlegroups.com
+    description: Runs the end-to-end test suite against a Kubernetes v1.18 cluster with all feature gates disabled
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+    preset-bazel-remote-cache-enabled: "true"
+    preset-bazel-scratch-dir: "true"
+    preset-cloudflare-credentials: "true"
+    preset-disable-all-alpha-beta-feature-gates: "true"
+    preset-venafi-cloud-credentials: "true"
+    preset-retry-flakey-tests: "true"
+    preset-ginkgo-focus-venafi: "true"
```

Comment:
With this preset, the test would run Venafi tests _only_ https://github.com/jetstack/testing/blob/master/config/jobs/cert-manager/config.yaml#L122-L125
```suggestion
    
```